cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -77,13 +78,16 @@ func TestSchedulerOptions(t *testing.T) {
 	// config file and kubeconfig
 	configFile := filepath.Join(tmpDir, "scheduler.yaml")
 	configKubeconfig := filepath.Join(tmpDir, "config.kubeconfig")
+	// On Windows, the paths contain backslashes, which could be interpreted by the yaml parser itself.
+	// If we don't escape them, we'd get yaml parsing errors.
+	escapedConfigKubeconfig := strings.Replace(configKubeconfig, `\`, `\\`, -1)
 	if err := os.WriteFile(configFile, []byte(fmt.Sprintf(`
 apiVersion: kubescheduler.config.k8s.io/v1beta3
 kind: KubeSchedulerConfiguration
 clientConnection:
   kubeconfig: "%s"
```

Comment:
I'm not sure if this actually works, but did you consider using single quotes instead?

Reply:
It works, the unit tests are passing, as you can see in this PR https://github.com/kubernetes/kubernetes/pull/110981, which includes these changes. In that PR we've ran the ``pull-ci-kubernetes-unit-windows`` job, in which we can see that these tests now pass.

Reply:
I'm asking if the test pass if you use single quote instead of double, without having to add code for escaping.

```yaml
  kubeConfig: '%s'
```

Reply:
this comment still needs an answer

Reply:
Indeed, it seems that escaping is not necessary if using ``kubeConfig: '%s'``.